test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -807,6 +806,7 @@ static void check_alg(odp_crypto_op_t op,
 
 	for (idx = 0; idx < count; idx++) {
 		int cipher_idx = -1, auth_idx = -1;
+		odp_bool_t bit_mode = ref[idx].length_is_in_bits;
```

Comment:
Since the bit_mode variable appeared only in the intermediate patches, I did not bother to rename it in v2. But I did rename cipher_bit_mode and auth_bit_mode that appeared in later patches.